src/components/UserDetailsTooltip/BaseUserDetailsTooltip/index.tsx

```diff
@@ -37,9 +37,12 @@ function BaseUserDetailsTooltip({accountID, fallbackUserDetails, icon, delegateA
     }
 
     let title = String(userDisplayName).trim() ? userDisplayName : '';
-    const subtitle = userLogin.trim() && LocalePhoneNumber.formatPhoneNumber(userLogin) !== userDisplayName ? Str.removeSMSDomain(userLogin) : '';
+    let subtitle = userLogin.trim() && LocalePhoneNumber.formatPhoneNumber(userLogin) !== userDisplayName ? Str.removeSMSDomain(userLogin) : '';
     if (icon && (icon.type === CONST.ICON_TYPE_WORKSPACE || !title)) {
         title = icon.name ?? '';
+        if (icon.type === CONST.ICON_TYPE_WORKSPACE) {
```

Comment:
I like it & I'm glad this works well, but can you please add a comment explaining why we are adding this? 🙏

Reply:
Thanks! And sorry but can you also add a little explanation of "why" that's needed?

Reply:
Thanks very much! 🙏